websites/docs-smoke-test/src/@commercetools-docs/gatsby-theme-docs/overrides/markdown-components.js

```diff
@@ -1,4 +1,15 @@
 import { ConnectedSquareIcon } from '@commercetools-uikit/icons';
 import addonComponents from './addon-components';
+import CardBannerOSSIcon from '../../../icons/card-banner-oss.svg';
+import CardBannerMcDevsIcon from '../../../icons/card-banner-mc-devs.svg';
+import CardBannerMcDocsIcon from '../../../icons/card-banner-mc-docs.svg';
+import CardBannerPlatformIcon from '../../../icons/card-banner-platform.svg';
 
-export default { ...addonComponents, DummyIcon: ConnectedSquareIcon };
+export default {
+  ...addonComponents,
+  DummyIcon: ConnectedSquareIcon,
+  CardBannerOSSIcon,
+  CardBannerMcDevsIcon,
+  CardBannerMcDocsIcon,
+  CardBannerPlatformIcon,
```

Comment:
I wouldn’t include the banner svgs in the core theme. For now they should be included and used only by the website itself. As far as I know, they are used only by the home website at the moment.

Reply:
Sorry never mind, I just realized this is the smoke test. 🤦‍♂️